doc/content/documentation/systems/Materials/ConcreteASREigenstrain.md

```diff
@@ -0,0 +1,265 @@
+\begin{equation}
+\label{eqn:asr_inc_rxn_extent}
+\begin{aligned}
+F(\Delta \xi) &= t_C(\theta^{t + \Delta t}, \xi^{t + \Delta t})\cdot \frac{\xi^{t + \Delta t} - \xi^t}{\Delta t} - 1 + \xi^{t + \Delta t}\\
+       &= t_C(\theta^{t + \Delta t}, \xi^t + \Delta\xi)\cdot \frac{\Delta\xi}{\Delta t} - 1 + (\xi^t + \Delta\xi)\\
+       &= 0
+\end{aligned}
+\end{equation}
+
+## Stress-dependent ASR volumetric strain $\Delta\varepsilon_{vol}^{ASR}$
+
+The procedure of [cite:saouma_constitutive_2006] to account for stress dependence in the volumetric strain resulting from the ASR reaction. Once the increment of ASR reaction extent $\Delta\xi$ is obtained, the ASR volumetric strain increment $\Delta\varepsilon_{vol}^{ASR}$ from time $t$ to $t+\Delta t$
+is then evaluated using the following formula
+\begin{equation}
+    \label{eqn:asr_volstrn}
+    \Delta\varepsilon_{vol}^{ASR} = \Gamma_t(f_t', \sigma_{I}|COD)\Gamma_c(\bar{\sigma}, f_c')g(H)\Delta\xi\varepsilon^{\infty}|_{\theta = \theta_0},
+\end{equation}
+where $f_t'$ is the tensile strength of the concrete; $\sigma_{I}$ is the maximum principal stress ( $>0$ under tensile stress);
+$COD$ is the crack opening displacement; $\bar{\sigma}$ is the ratio between the hydrostatic stress and compressive strength of concrete, and
+ $\varepsilon^{\infty}$ is the laboratory-determined maximum free volumetric expansion at the reference temperature $\theta_0$.
+
+The function $g(H)$ in [eqn:asr_volstrn] accounts for the dependency of gel expansion on the water in concrete and takes the form:
+\begin{equation}
+    g(H) = H^m,
+\end{equation}
+where the exponent $m$ is an empirical constant and $H$ is the relative humidity in concrete.
+$g(H)$ has a value between 0 and 1. For studies of concrete dams, one can reasonably assume $g(H)$ has a value of
+1. For other concrete structures, however, $g(H)$ has to be determined by either lab experiments or parameter fitting.
+
+Function $\Gamma_t(f_t', \sigma_{I}|COD)$ in [eqn:asr_volstrn] accounts for the reduction of ASR expansion due to tensile cracking
+(e.g., gels are adsorbed into tensile macrocracks), and takes the following form:
+\begin{equation}
+    \Gamma_t = \left\{\begin{array}{lll}
+            \text{Elasticity}  & \begin{array}{lll}
+                    1 & if & (\sigma_{I} \le \gamma_t f_t') \\
+                    \Gamma_r + (1 - \Gamma_r)\frac{\gamma_t f_t'}{\sigma_{I}} & if & (\gamma_t f_t' \le \sigma_{I}) \\
+                    \end{array} \\\\
+            \text{Smeared crack} & \begin{array}{lll}
+                    1 & if & COD_{max} \le \gamma_t f_t' < \sigma_{I} \\
+                    \Gamma_r + (1 - \Gamma_r)\frac{\gamma_t W_c}{COD_{max}} & if & Y_t W_c < COD_{max}
+                    \end{array}
+        \end{array}\right.
+\end{equation}
+in which $\gamma_t$ is the fraction of the tensile strength beyond which gel is adsorbed by cracks and is a user input parameter in simulation, and
```

Comment:
I would drop the `Smeared crack` part of the equation as it is currently not implemented.